### ISSUE TITLE
[Bug] Links in bio

### ISSUE THREAD

**Issue by neilideal** [sk22/megalodon#118]
I don't know if this is a bug of Megalodon or Mastodon, but links in profile descriptions, even if they are specified with @, open in the browser, while in posts they open as profiles in the application.

Link with video: https://drive.google.com/file/d/1DfzRn9_fLuMZW3csmhLsg0zEdfs2Ss7J/view?usp=drivesdk

**Comment by sk22** [comment]
yes, that's an upstream bug (it'd be great if you could try that out using the provided upstream apk in the release before creating the issue here). there's some similar issues on the upstream repo (mostly about fedi links not opening directly in the app - except for ones that point to the home instance), but i didn't find anything about @-handles not working in bios. would probably be good if you could create the issue there!

**Comment by sk22** [comment]
also, it's not something i'd like to fix here - if it bugs me a lot, i might end up fixing it and creating a pull request, which would probably get merged upstream too, but i don't really see it in my responsibility to do that. hence closing this issue for now!
